### PR TITLE
Signup login pages

### PR DESCRIPTION
**Pull request by cl-codes**
### What this PR does (required):
- Style updates to login and sign-up page

### Screenshots / Videos (front-end only):
screenshot - sign up
Login

### Any issues with the current functionality (optional):
Some challenges with overriding MUI default styles, so used `sx={{}}` on a one-off basis where required.

There are some adjustments I would have liked to make, but wasn't able to get working given time constraint:
  1. Input field text and border colors should be lightened, font style also to be tweaked.
  2. I added FormLabel elements for the "look", but haven't figured out how to connect them with the related inputs for accessibility (like they were in the original template).
  3. Grid outline/edges are too dark (especially the bottom), should be smoother or no outline.

### CODE REVIEW

File: client/src/components/AuthHeader/AuthHeader.tsx [hunk 1]
```diff
@@ -17,15 +17,22 @@ const AuthFooter = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
     <Box
       p={1}
       display="flex"
-      justifyContent="flex-end"
+      justifyContent="center"
       alignSelf="flex-end"
-      marginRight={5}
+      marginRight={0}
```
Comment (moffatethan) [inline client/src/components/AuthHeader/AuthHeader.tsx:22]: All of these could go into the `sx` property. 😃

File: client/src/pages/Login/Login.tsx [hunk 2]
```diff
@@ -37,24 +37,46 @@ export default function Login(): JSX.Element {
...
+                  component="h1"
+                  variant="h5"
+                  align="center"
+                >
                   Welcome back!
                 </Typography>
               </Grid>
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
+            <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
```
Comment (moffatethan) [inline client/src/pages/Login/Login.tsx:79]: We should probably rename this component to AuthFooter now since it's on the bottom.

**Review by moffatethan — CHANGES_REQUESTED** [review]
Some minor changes Crystal! Overall, clean code. Would have liked to see more use of `sx` but I'm not going to request those changes due to it not being super critical. 

Going forward we can preference `sx` for things like `margins`, `padding`, `flex`, etc.

After those changes are addressed just request a second review and I'll approve. 🚀 💯

**Comment by moffatethan** [comment]
Also, good matching the mocks!

File: client/src/components/AuthHeader/AuthHeader.tsx (hunk 1 above)
Comment (cl-codes) [reply client/src/components/AuthHeader/AuthHeader.tsx]: moved - thanks!

File: client/src/pages/Login/Login.tsx (hunk 2 above)
Comment (cl-codes) [reply client/src/pages/Login/Login.tsx]: renamed files/references

**Review by moffatethan — APPROVED** [review]
Good to go 🚀 💯

**Pull request merged**